x-pack/plugins/endpoint/server/routes/metadata/query_builders.ts

```diff
@@ -4,14 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { KibanaRequest } from 'kibana/server';
-import { EndpointAppConstants } from '../../../common/types';
+import { EndpointAppConstants, MetadataIndexGetBodyResult } from '../../../common/types';
 import { EndpointAppContext } from '../../types';
 import { esKuery } from '../../../../../../src/plugins/data/server';
 
+/**
+ * TODO, add comment to exported function.
+ */
 export const kibanaRequestToMetadataListESQuery = async (
-  request: KibanaRequest<any, any, any>,
+  request: KibanaRequest<unknown, unknown, MetadataIndexGetBodyResult>,
   endpointAppContext: EndpointAppContext
-): Promise<Record<string, any>> => {
+): Promise<Record<string, unknown> & { size: number; from: number }> => {
```

Comment:
A consumer of this function requires this, and this function does in fact return this.